README.md

```diff
@@ -40,6 +43,7 @@ To be eligible for delegation, such validators must:
   its intended delegations
 - have bonded more than 50% of the ATOMs won during Game of Stakes to their
   validator
+- not be composed of any personnel who is/was a member of Cosmos/Tendermint team
```

Comment:
This was already a requirement for GoS winners so no need to re-iterate here.

Reply:
Agree!